src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,104 @@
+import styled from 'styled-components/macro'
+
+import Button from '../Button/Button'
+
+import 'css.gg/icons/css/film.css'
+
+export default function MovieCard({
+  onHandleDelete,
+  key,
+  movieName,
+  movieFoodCategory,
+  display,
+  right,
+}) {
+  return (
+    <>
```

Comment:
No need for the fragments here as `MovieWrapper` is wrapping around all the other elements